Makefile

```diff
@@ -32,46 +34,64 @@ clean:
 
 .PHONY: devkit
 devkit:
-	if $$(docker images | grep mesosphere/adminrouter-devkit | grep -q latest); then \
+	if $$(docker images | grep mesosphere/adminrouter-devkit | grep -q full); then \
 		echo "+ Devkit image already build"; \
 	else \
 		echo "+ Building devkit image"; \
-		docker rmi -f mesosphere/adminrouter-devkit:latest; \
 		docker build \
 			--rm --force-rm \
-			-t \
-			mesosphere/adminrouter-devkit:latest ./docker/ ;\
+			-t mesosphere/adminrouter-devkit:noresty \
+			-f ./docker/Dockerfile \
+				./docker/ && \
+		docker run \
```

Comment:
Yes, I am sorry - it was late, I was tired and just mixed the answers up :(

Regarding this question - in order to keep the same buildchain for both DC/OS build and local/laptop development we need to rely on tagging containers:
* during DC/OS build, the `build` script is launched inside the adminrouter-devkit container https://github.com/mesosphere/dcos-enterprise/pull/637/files#diff-e5cec6b71cde7c47f90f9173d80309f4R11. Building new containers is unavailable there.
* local development launches the same script, but with different params: https://github.com/dcos/adminrouter/pull/53/files#diff-b67911656ef5d18c4ae36cb6741b7965R49 We could build a new container here, but we would still have keep the current approach for DC/OS build.

I hope that this clears things up a bit :)